autotest/src/tests/test_shm.c

```diff
@@ -66,6 +66,26 @@ void test_shm_invalidmap(void) {
     TEST_END();
 }
 
+void test_shm_infos(void) {
+    Status res;
+    shmh_t shm;
+    shm_infos_t infos;
+    TEST_START();
+    res = sys_get_shm_handle(shms[0].id);
```

Comment:
with this using dts at app level is no more needed for SHM. The effective goal is to manipulate DTS only in OS libraries, so that application (including drivers) never need to add jinja code. For drivers this will be done using the driver framework we are working on, as a easy device driver usual dependency, for SHM & DMA streams, no need for DTS parsing at app level ideally.